winforms/src/toga_winforms/widgets/multilinetextinput.py

```diff
@@ -102,3 +112,10 @@ def scroll_to_bottom(self):
     def scroll_to_top(self):
         self.native.SelectionStart = 0
         self.native.ScrollToCaret()
+
+    def _text_changed(self):
+        # Check if the text has changed since the last time we checked.
+        # This is used to avoid triggering the on_change handler when the
+        # Winforms.RichFieldBox fires TextChange event due to style changes such
+        # as font, color, text alignment, etc.
+        return True if self.native.Text != self._text_history else False
```

Comment:
The result of a boolean comparison will already be boolean, so there's no need to wrap this in an if. This is why I suggested a separate method wasn't needed - the thing being abstracted is a single binary operation.
```suggestion
        return self.native.Text != self._text_history
```